rhttpc-amqp-jdbc/src/main/scala/rhttpc/transport/amqpjdbc/slick/AddingPropertiesToScheduledMessagesMigration.scala

```diff
@@ -25,10 +25,12 @@ import slick.sql.SqlProfile.ColumnOption.NotNull
 trait AddingPropertiesToScheduledMessagesMigration extends SlickJdbcMigration {
   import profile.api._
 
-  private lazy val sheduledMessagesWithoutPropsMigration = new CreatingScheduledMessagesTableMigration {
+  class V1_001__CreatingScheduledMessagesTableMigration extends CreatingScheduledMessagesTableMigration {
```

Comment:
Not sure about this one, as well as similar inner class `V1_001__AddingPropertiesToScheduledMessagesMigration`. I used `BaseJavaMigration` for `SlickJdbcMigration` that requires that naming format, so anonymous function didn't work.